Cargo.toml

```diff
@@ -15,7 +15,7 @@ edition = "2018"
 vendored-openssl = ["ssh2/vendored-openssl"]
 
 [dependencies]
-ssh2 = "0.7"
+ssh2 = {git = "https://github.com/alexcrichton/ssh2-rs", branch = "send_safe" }
```

Comment:
I think adding the dependency on the branch here would make the [patch.crates-io] section irrelevant.